From patchwork Fri Jun 24 02:13:33 2016
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [v5,8/8] iommu/rockchip: Enable Rockchip IOMMU on ARM64
From: Shunqian Zheng <zhengsq@rock-chips.com>
X-Patchwork-Id: 9196373
Message-Id: <1466734413-7453-9-git-send-email-zhengsq@rock-chips.com>
To: joro@8bytes.org, heiko@sntech.de, robh+dt@kernel.org,
 mark.rutland@arm.com, 
 linux@armlinux.org.uk, mark.yao@rock-chips.com, airlied@linux.ie,
 tfiga@google.com, xxm@rock-chips.com
Cc: devicetree@vger.kernel.org, linux-kernel@vger.kernel.org,
 dri-devel@lists.freedesktop.org, Tomasz Figa <tfiga@chromium.org>,
 linux-rockchip@lists.infradead.org, iommu@lists.linux-foundation.org, 
 Shunqian Zheng <zhengsq@rock-chips.com>,
 linux-arm-kernel@lists.infradead.org
Date: Fri, 24 Jun 2016 10:13:33 +0800

From: Simon Xue <xxm@rock-chips.com>

This patch makes it possible to compile the rockchip-iommu driver on
ARM64, so that it can be used with 64-bit SoCs equipped with this type
of IOMMU.

Signed-off-by: Simon Xue <xxm@rock-chips.com>
Signed-off-by: Shunqian Zheng <zhengsq@rock-chips.com>
Signed-off-by: Tomasz Figa <tfiga@chromium.org>
---
 drivers/iommu/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/iommu/Kconfig b/drivers/iommu/Kconfig
index ad08603..5572621 100644
--- a/drivers/iommu/Kconfig
+++ b/drivers/iommu/Kconfig
@@ -218,7 +218,7 @@ config OMAP_IOMMU_DEBUG
 
 config ROCKCHIP_IOMMU
 	bool "Rockchip IOMMU Support"
-	depends on ARM
+	depends on ARM || ARM64
 	depends on ARCH_ROCKCHIP || COMPILE_TEST
 	select IOMMU_API
 	select ARM_DMA_USE_IOMMU
